app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -60,7 +60,7 @@ To be eligible for the IRP, you must be coming to England to teach or train to t
 
 You already have the right to train or work as a teacher in England if you're in the UK on (one of the following):
 
-* [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa)
+* [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa) – please note this scheme shut to new applications on 19 February 2024
```

Comment:
@laurahermionetennant92 feel free to ignore! But I wondered if is closed might be better than shut to match the wording that's used on the gov.uk page it links to?

Reply:
yes good point